### PR TITLE
Make README.md code examples compile without errors

### PR DESCRIPTION
**Pull request by petertodd**

### CODE REVIEW

**Review by evanlinjin — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -96,14 +97,15 @@ use bdk::electrum_client::Client;
 use bdk::wallet::AddressIndex::New;
 
 use base64;
-use bitcoin::consensus::serialize;
+use bdk::bitcoin::consensus::serialize;
```
Comment (danielabrozzoni) [inline README.md:100]: Why is this needed? The example runs just fine for me without it

**Comment by petertodd** [comment]
On December 13, 2022 2:01:07 AM CST, Daniela Brozzoni ***@***.***> wrote:
>@danielabrozzoni commented on this pull request.
>
>
>
>> @@ -96,14 +97,15 @@ use bdk::electrum_client::Client;
> use bdk::wallet::AddressIndex::New;
> 
> use base64;
>-use bitcoin::consensus::serialize;
>+use bdk::bitcoin::consensus::serialize;
>
>Why is this needed? The example runs just fine for me without it
.
Because you have `bitcoin` as a dependency as well as `bdk`; with my change you can run the example with just `bdk` in your `Cargo.toml`

**Review by danielabrozzoni — APPROVED** [review]
> Because you have `bitcoin` as a dependency as well as `bdk`; with my change you can run the example with just `bdk` in your `Cargo.toml`

Oh right, I noticed that but I misread that line and thought you were introducing a new import 😅 

ACK 99e703714fd8a489c8f9969dc6c91b2124558fa2

**Comment by notmandatory** [comment]
Looks like this is ready to go after I get #822 in to fix the nightly docs CI error.

**Comment by danielabrozzoni** [comment]
I merged #822, try rebasing plz :) (I can't merge until CI passes)

**Comment by petertodd** [comment]
On Wed, Dec 14, 2022 at 11:19:46AM -0800, Daniela Brozzoni wrote:
> I merged #822, try rebasing plz :) (I can't merge until CI passes)

Rebased

**Comment by danielabrozzoni** [comment]
ACK d1e5b87bfc1d37c5cd6631427ab521a59fa4e67a
